Bottom: 18f33e3948f3df1d49fce1436162a2a26d94cfdf
Top:    3456e92b3ec5055a26635ad8ec9654510e13c6f3
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-05-17 18:08:43 +0100

Disable process limits


---

diff --git a/src/polygon.c b/src/polygon.c
index 420772b..fb40faf 100644
--- a/src/polygon.c
+++ b/src/polygon.c
@@ -132,6 +132,7 @@ polygon_init (void)
 
   bw_rotate_circle_seg[0] =  cos_ang;  bw_rotate_circle_seg[1] =  sin_ang;
   bw_rotate_circle_seg[2] = -sin_ang;  bw_rotate_circle_seg[3] =  cos_ang;
+return;
 
   /* DEBUG - AVOID PCB running the system out of memory! */
   getrlimit (RLIMIT_AS, &limit);
